packages/documentation/src/pages/forms/form-tutorial-advanced.mdx

```diff
@@ -1,19 +1,15 @@
 ---
-title: VAFS - advanced tutorial
+title: Using common definitions
```

Comment:
This isn't what this page is about. Using common definitions is one of a few things talked about here, not the focus of the entire page.